servers/dsub/jobs/controllers/capabilities_controller.py

```diff
@@ -1,7 +1,43 @@
+from jobs.controllers.utils.providers import ProviderType
+from jobs.models.authentication_capability import AuthenticationCapability
+from jobs.models.capabilities_response import CapabilitiesResponse
+
+
 def get_capabilities():
     """Get the capabilities of this backend implementation.
 
     Returns:
         CapabilitiesResponse: Response containing this backend's capabilities
     """
-    raise NotImplemented('Not yet supported.')
+    capabilities = CapabilitiesResponse(
+        authentication=AuthenticationCapability(
+            is_required=true,
+            scopes=[
+                'https://www.googleapis.com/auth/genomics',
+                'https://www.googleapis.com/auth/cloudplatformprojects.readonly'
+            ]),
+        display_fields={
+            'name': 'Job',
+            'status': 'Status',
+            'submission': 'Submitted',
+            'labels.job_id': 'Job ID',
+            'labels.task_id': 'Task ID',
+            'extensions.user_id': 'User ID',
```

Comment:
This should be camelCased, I think. Here and throughout

Reply:
That's a good point since the typescript models are camelCase. Updated, I guess we leave labels snake_case right?

Reply:
Actually, they should be hyphenated. It won't be by style convention here - it will need to exactly match the casing of the actual label keys we expect to be returning.

Reply:
Right, fixed.